bundle/edu.gemini.p1backend/TODO.md

```diff
@@ -0,0 +1,50 @@
+# Short term
+
+Something that improves the situation now. Nothing for the future. A dead-end by
+design. Something to keep chugging away until we do it right. It shouldn't
+take more work than 2 weeks.
+
+1. Code lives in ocs.
+1. URLs:
+   - Receiver of P1 submissions: phase1.gemini.edu/receive/.../...
+   - Viewer of P1 submissions: phase1.gemini.edu/fetch/.../...
+1. Each backend a different path like now.
+1. Configuration file for each backend (that file is read with every new
+   submission). Used for prepending semester to Ids, close deadline, etc.
+1. Id counter from previous file names (hashes discussed but discarded).
+1. Different backend for joints? If not possible use hard links.
+1. Generate PDFs.
+1. Stick together the XML and PDFs in tarballs.
+1. Send notifications *after* everything is successful. Consult configuration
+   file for emails.
+1. Multiple p1 versions:
```

Comment:
This is not correct. We cannot upgrade proposals from older PITs on the backend. each backend needs a semester/version and will only accept that version. The tricky part here is that we should accept multiple versions. Given we can't have multiple versions of model.p1 probably we need to have multiple backends with a front end.